tapalcatl_server/server.go

```diff
@@ -253,15 +253,28 @@ func (bm *OnDemandBufferManager) Get() *bytes.Buffer {
 func (bm *OnDemandBufferManager) Put(buf *bytes.Buffer) {
 }
 
+func logFatalCfgErr(logger JsonLogger, msg string, xs ...interface{}) {
+	logger.Error(LogCategory_ConfigError, msg, xs...)
+	os.Exit(1)
+}
+
 func main() {
 	var listen, healthcheck, debugHost string
 	var poolNumEntries, poolEntrySize int
 	var metricsStatsdAddr, metricsStatsdPrefix string
 
 	hc := handlerConfig{}
 
+	systemLogger := log.New(os.Stdout, "", log.LstdFlags|log.LUTC|log.Lmicroseconds)
+	hostname, err := os.Hostname()
```

Comment:
That's exactly what we get from opsworks, and is what I would want as the hostname too. What were you expecting? What other information from the stack were you thinking of adding that could be useful?

I tried it on a docker instance on an aws deployment, and got an opaque identifier like `cce48f09e9d3`, which matched the container id.

The only thing I personally care about here is that it's a unique identifier that we can use to filter the logs down to just that particular instance. The hostname seems to serve that purpose, regardless of what that is.